model/src/constants.F90

```diff
@@ -20,7 +20,7 @@ MODULE CONSTANTS
 !/
 !/    Copyright 2009-2012 National Weather Service (NWS),
 !/       National Oceanic and Atmospheric Administration.  All rights
-!/       reserved.  WAVEWATCH III is a trademark of the NWS. 
+!/       reserved.  WAVEWATCH III is a trademark of the NWS.
```

Comment:
This entire file is just white space changes.  Should this go ahead and be reverted?

Reply:
This is stripping of trailing white space. Trailing whitespace should not be committed.

Reply:
The preference is to take care of trailing white spaces in it's own PR to the develop branch instead of being combined here.  While github does have a great feature to ignore white spaces in a review, it's still preferred to handle this separately.